api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterSpec.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.strimzi.api.kafka.model.template.KafkaExporterTemplate;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Representation of the Kafka Exporter deployment.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "image", "groupRegex",
+        "topicRegex", "resources", "logging",
+        "enableSaramaLogging", "template"})
+@EqualsAndHashCode
+public class KafkaExporterSpec implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private String image;
+    private String groupRegex = ".*";
```

Comment:
could be more explicit calling it `consumerGroupRegex`? The same for the related getter/setter?

Reply:
I named this similarly to the flags in the Kafka Exporter project are named https://github.com/danielqsj/kafka_exporter#flags

Reply:
fine with that then